### PR TITLE
chromaprint build fix

### PR DESCRIPTION
**Pull request by th0ma7**
_Motivation:_  Since migration to `cmake` helpers `chromaprint` does not build anymore.  Intent is to solve that issue while keeping linkage to `ffmpeg`.
_Linked issues:_  https://github.com/acoustid/chromaprint/issues/79

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

Sub-builds checks:
- [x] cross/libwebsockets
- [x] cross/intel-media-driver
- [x] cross/intel-media-sdk
- [x] cross/libmysqlclient
- [x] cross/libaom
- [x] cross/gmmlib
- [x] cross/chromaprint
- [x] cross/umurmur
- [x] cross/soxr
- [x] cross/svt-av1
- [x] cross/svt-hevc
- [x] cross/uchardet

### CODE REVIEW

File: mk/spksrc.cross-cmake-env.mk [hunk 1]
```diff
@@ -6,11 +6,8 @@ CMAKE_ARGS += -DCMAKE_SYSTEM_NAME=Linux
 CMAKE_ARGS += -DCMAKE_BUILD_TYPE=Release
 CMAKE_ARGS += -DCMAKE_C_COMPILER=$(TC_PATH)$(TC_PREFIX)gcc
 CMAKE_ARGS += -DCMAKE_CXX_COMPILER=$(TC_PATH)$(TC_PREFIX)g++
-CMAKE_ARGS += -DCMAKE_EXE_LINKER_FLAGS=-L$(INSTALL_DIR)$(INSTALL_PREFIX)/lib -Wl,--rpath-link,$(INSTALL_DIR)$(INSTALL_PREFIX)/lib -Wl,--rpath,$(INSTALL_PREFIX)/lib
 CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH=$(INSTALL_DIR)$(INSTALL_PREFIX)
 CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER
-CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY
-CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY
```
Comment (ymartin59) [inline mk/spksrc.cross-cmake-env.mk:13]: Isn't there a risk other packages built with cmake support may no longer work? I would give it a check...

**Comment by ymartin59** [comment]
@th0ma7 armv7 builds have fail... something to check too

**Comment by th0ma7** [comment]
> @th0ma7 armv7 builds have fail... something to check too

Yeah, armv7 and other failures have all related to download failures:
```
2020-08-05T12:58:11.9115015Z wget https://gmplib.org/download/gmp/gmp-6.1.2.tar.lz
2020-08-05T13:00:22.2950088Z failed: Connection timed out.
2020-08-05T13:02:33.3670761Z failed: Connection timed out.
2020-08-05T13:04:46.4870198Z failed: Connection timed out.
2020-08-05T13:06:59.6070674Z failed: Connection timed out.
2020-08-05T13:09:14.7751315Z failed: Connection timed out.
2020-08-05T13:11:29.9430660Z failed: Connection timed out.
2020-08-05T13:13:45.1187932Z failed: Connection timed out.
2020-08-05T13:16:02.3269972Z failed: Connection timed out.
2020-08-05T13:18:19.8135517Z failed: Connection timed out.
2020-08-05T13:20:38.8071394Z failed: Connection timed out.
2020-08-05T13:22:58.0710807Z failed: Connection timed out.
2020-08-05T13:25:17.3350395Z failed: Connection timed out.
2020-08-05T13:27:36.5989853Z failed: Connection timed out.
2020-08-05T13:29:55.8631570Z failed: Connection timed out.
2020-08-05T13:32:15.1270066Z failed: Connection timed out.
2020-08-05T13:34:34.3912289Z failed: Connection timed out.
2020-08-05T13:36:53.6553081Z failed: Connection timed out.
2020-08-05T13:39:12.9191086Z failed: Connection timed out.
2020-08-05T13:41:32.1830058Z failed: Connection timed out.
2020-08-05T13:43:51.4473041Z failed: Connection timed out.
2020-08-05T13:43:51.4478036Z make[4]: *** [../../mk/spksrc.download.mk:57: download_target] Error 4
2020-08-05T13:43:51.4478929Z make[4]: Leaving directory '/github/workspace/cross/gmp'
2020-08-05T13:43:51.4486813Z make[3]: *** [../../mk/spksrc.depend.mk:44: depend_target] Error 2
```

File: mk/spksrc.cross-cmake-env.mk (hunk 1 above)
Comment (th0ma7) [reply mk/spksrc.cross-cmake-env.mk:13]: Good catch.  My bets are that it doesn't but i will run a quick check to confirm all other packages migrated to cmake helper are ok based on PR https://github.com/SynoCommunity/spksrc/pull/3998

Comment (th0ma7) [reply mk/spksrc.cross-cmake-env.mk:13]: @ymartin59 all non-ffmpeg related builds passed, all good.

**Comment by ymartin59** [comment]
@th0ma7 You have to test commands in package at least on one system (for instance xpenology virtual machine)
```
$ fpcalc --version
fpcalc: error while loading shared libraries: libchromaprint.so.1: cannot open shared object file: No such file or directory
```

**Comment by th0ma7** [comment]
I think I've found the issue.
I had tested it just before switching to shared lib without updating the `PLIST` ...

**Pull request merged**